lib/server.js

```diff
@@ -129,6 +129,30 @@ function multipleBindings (mainServer, httpHandler, serverOpts, listenOptions, o
 
             if (!_ignoreErr) {
               this[kServerBindings].push(secondaryServer)
+              /**
+               * Due to the nature of the feature is not possible to know
+               * ahead of time the number of bindings that will be made
+               * For instance, each binding is hooked to be closed at their own
+               * peace through the `onClose` hook.
+               * It also allows them to handle possible connections already
+               * attached to them if any
+               */
+              this.onClose((_, done) => {
+                if (this[kState].listening) {
+                  // No new TCP connections are accepted
+                  secondaryServer.close(done)
+                  /* istanbul ignore next: Cannot test this without Node.js core support */
+                  if (serverOpts.forceCloseConnections === 'idle') {
+                    // Not needed in Node 19
+                    secondaryServer.closeIdleConnections()
+                  /* istanbul ignore next: Cannot test this without Node.js core support */
+                  } else if (typeof secondaryServer.closeAllConnections === 'function' && serverOpts.forceCloseConnections) {
+                    secondaryServer.closeAllConnections()
+                  }
+                } else {
+                  done(null)
```

Comment:
we may shallow an error here?

```suggestion
                  done(_)
```

Reply:
Good catch 👍